bcipy/signal/model/pca_rda_kde/pca_rda_kde.py

```diff
@@ -0,0 +1,126 @@
+import pickle
+from pathlib import Path
+from typing import List
+
+import numpy as np
+from bcipy.signal.model.base_model import ModelEvaluationReport, SignalModel
+from bcipy.signal.model.pca_rda_kde.classifier import RegularizedDiscriminantAnalysis
+from bcipy.signal.model.pca_rda_kde.cross_validation import cost_cross_validation_auc, cross_validation
+from bcipy.signal.model.pca_rda_kde.density_estimation import KernelDensityEstimate
+from bcipy.signal.model.pca_rda_kde.dimensionality_reduction import ChannelWisePrincipalComponentAnalysis
+from bcipy.signal.model.pca_rda_kde.pipeline import Pipeline
+from bcipy.signal.exceptions import SignalException
+
+
+class PcaRdaKdeModel(SignalModel):
+    def __init__(self, k_folds=10):
+        self.k_folds = k_folds
+        self.model = None
+        self._ready_to_predict = False
+
+    def fit(self, train_data: np.array, train_labels: np.array) -> SignalModel:
+        """
+        Train on provided data using K-fold cross validation and return self.
+
+        Parameters:
+            train_data: shape (Channels, Trials, Trial_length) preprocessed data
+            train_labels: shape (Trials,) binary labels
+
+        Returns:
+            trained likelihood model
+        """
+        model = Pipeline([ChannelWisePrincipalComponentAnalysis(var_tol=1e-5, num_ch=train_data.shape[0]),
```

Comment:
We could pull up these hardcoded values to make it easier to change in the future. Not sure if it's worth it.

Reply:
I'd rather leave for now if that's OK

Reply:
(just out of laziness)